addons/events/src/components/Event.tsx

```diff
@@ -67,7 +76,7 @@ const Label = styled.label({
   textAlign: 'right',
   width: 100,
   fontWeight: '600',
-});
+} as any);
```

Comment:
This one also gives me errors and I couldn't resolve it. Error goes like this 
```
Argument of type '() => { display: string; boxSizing: "border-box"; verticalAlign: string; paddingRight: number; paddingTop: number; textAlign: "right"; width: number; fontWeight: "600"; }' is not assignable to parameter of type 'TemplateStringsArray'.
  Type '() => { display: string; boxSizing: "border-box"; verticalAlign: string; paddingRight: number; paddingTop: number; textAlign: "right"; width: number; fontWeight: "600"; }' is missing the following properties from type 'TemplateStringsArray': raw, concat, join, slice, and 16 more.ts(2345)
```
Changing code like this also didn't work
```
const Label = styled.label(() => ({
  display: 'table-cell',
  boxSizing: 'border-box',
  verticalAlign: 'top',
  paddingRight: 5,
  paddingTop: 7,
  textAlign: 'right',
  width: 100,
  fontWeight: '600',
}));
```

Similar code is already used at other parts
https://github.com/storybookjs/storybook/blob/8aae29d55234b3267cfd131d2bf94e1ef44b9578/addons/a11y/src/components/Tabs.tsx#L18-L24
https://github.com/storybookjs/storybook/blob/2887028ad221f4641d5cc7c33cb1269cc2bf1d01/lib/components/src/form/field/field.tsx#L4-L13

Here are for more information [this](https://github.com/emotion-js/emotion/issues/1298),  and [this](https://spectrum.chat/emotion/help/emotion-typescript-debugging~df93d2c1-d607-4ff5-80e0-3914219ad8ea?authed=true#)